scripts/populateDAO.ts

```diff
@@ -0,0 +1,335 @@
+                amount: Util.toWei(amount)
+            })
+            logTx(network, tx);
+        }
+
+        async function approve() {
+            log(Level.Info, `Pre-approving 1000 GEN for current account`);
+            const {tx} = await stakingToken.approve({spender: gp.address, amount: Util.toWei(1000)});
+            approved[web3.eth.defaultAccount] = true;
+            logTx(network, tx);
+        }
+        await approve();
+
+        for (i = 0 ; i < steps.length; i++) {
+            const step = steps[i];
+            switch (step.action) {
+                case Action.SwitchAccount:
+                    const { id } = step.args;
+                    if (id < 0 || id >= accounts.length) {
+                        throw new Error(`Account id ${id} cannot be found (maximum id can be ${accounts.length - 1})`)
+                    }
+                    log(Level.Info, `Switching to account #${id}: ${accounts[id]}`);
+                    web3.eth.defaultAccount = accounts[id];
+
+                    if (!approved[web3.eth.defaultAccount]) {
+                        await approve();
+                    }
+                    break;
+                case Action.CreateProposal:
+                    if (mapping[step.args.id]) {
+                        throw new Error(`The id ${step.args.id} was already used for another proposal`)
+                    }
+                    const proposalId = await createProposal(step.args)
+                    mapping[step.args.id] = proposalId;
+                    break;
+                case Action.Vote:
+                    if (!mapping[step.args.id]) {
+                        throw new Error(`No proposal with id ${step.args.id} was created`)
+                    }
+                    await vote({...step.args, proposalId: mapping[step.args.id]})
+                    break;
+                case Action.Stake:
+                    if (!mapping[step.args.id]) {
+                        throw new Error(`No proposal with id ${step.args.id} was created`)
+                    }
+                    await stake({...step.args, proposalId: mapping[step.args.id]})
+                    break;
+            }
+        }
+        log(Level.Success, 'Done.');
+        process.exit(0)
+    } catch (e) {
+        log(Level.Error, `An error occured during script execution: ${e.stack}`);
+        process.exit(1)
+    }
+}
+
+const argv: any =
+    yargs
+        .config(
+            'script',
+            'path to a script file',
+            (path) => JSON.parse(fs.readFileSync(path, 'utf-8'))
+        )
+        .option(
+            'logfile', {
+                alias: 'l',
+                describe: 'path to a log file to save results',
+                type: 'string'
+            }
+        )
+        .option(
+            'api', {
+                alias: 'a',
+                describe: 'api root url',
+                default: 'https://daostack-alchemy.herokuapp.com',
```

Comment:
hmm, i think we should not default to production server, lets default to daostack-alchemy-server-stage

Reply:
though really we just need to always use the right server for the network used, im going to pull that ticket in